src/components/button/icons/user-icon.jsx

```diff
@@ -0,0 +1,16 @@
+const UserIcon = () => (
+  <svg
+    xmlns="http://www.w3.org/2000/svg"
+    viewBox="0 0 200 200"
+    xmlSpace="preserve"
+    width="50"
+    height="50"
```

Comment:
This is incorrect. You should either use `48px` or use tailwind classes. Generally we keep font sizes to be in multiple of 4 and also it should have a correct unit.

Reply:
using className of tailwind css to style it. thanks for your suggestion.